css/main.css

```
@@ -1,3 +1,498 @@
+    background-color: var(--lightBeige);
+    display:flex;
+    flex-direction: column;
+    align-items: center;
+    gap:20px;   
+    box-shadow: -7px 7px 15px 2px #939393;
+    position:relative ; 
+    top: -300px; 
+ 
+}   
+      
+.personal-image{
+    height: 250px;
+    width: 250px;
+    padding: 20px;
+    border-radius: 100%;
+    margin-top: 20px;
+}   
+.divider{
+    border-color: var(--royalBlue);
+    width: 68px;
+} 
+.name-family{
+    font-size: 21px;
+    line-height: 1.2em;
+}
+.resume{
+        
+    background-color: var(--royalBlue);
+    border: 2px solid var(--royalBlue);
+    color: var(--white);
+    padding: 5px 35px;
+    border-radius: 20px;
+    font-family: 'Poppins', sans-serif;
+}
+.resume:hover{
+    background-color: var(--lightBeige);
+    border: 2px solid var(--Black);
+    color: var(--Black);
+
+}
+.project{
+        
+    background-color: var(--lightBeige);
+    border: 2px solid var(--Black);
+    color: var(--Black);
+    padding: 5px 35px;
+    border-radius: 20px;
+    font-family: 'Poppins', sans-serif;
+}
+.project:hover{
+    background-color: var(--royalBlue);
+    border: 2px solid var(--royalBlue);
+    color: var(--white);
+
+}
+.buttun-personal{
+    display: none;
+}  
+.buttun-professional{
+    margin-bottom: 30px;
+    display: flex;
+    gap: 20px;
+}
+
+
+
+  
+footer{       
+    margin:5px 10px 60px 20px ;
+    display: flex;
+    flex-direction: column;
+    gap:30px;
+    
+}
+
+
+.contact{
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+}
+.personal_introduction {
+    
+    margin: 20px;
+    display: flex;
+    flex-direction: column;
+    row-gap: 10px;
+    position: absolute;
+    bottom: 1px;
+    
+    
+
+}
+
+
+#menuToggle
+{
+  display: block;
+  position: relative;
+  top: 50px;
+  
+  
+  
+  
+  z-index: 1;
+  
+  -webkit-user-select: none;
+  user-select: none;
+}
+
+#menuToggle a
+{
+  text-decoration: none;
+  color: #232323;
+  
+  transition: color 0.3s ease;
+}
+
+#menuToggle a:hover
+{
+  color:var(--royalBlue);
+}
+
+
+#menuToggle input
+{
+  display: block;
+  width: 40px;
+  height: 32px;
+  position: absolute;
+  top: -7px;
+  left: -5px;   
+  cursor: pointer;  
+  opacity: 0; /* hide this */
+  z-index: 2; /* and place it over the hamburger */  
+  -webkit-touch-callout: none;
+}
+
+/*
+ * Just a quick hamburger
+ */
+.span-1
+{
+  display: block;
+  width: 30px;
+  height: 4px;
+  margin-bottom: 5px;
+  position: relative;
+  
+  background:var(--royalBlue);
+  border-radius: 3px;
+  z-index: 1;
+  bottom: 30px;
+  
+  
+}
+.span-2
+{
+  display: block;
+  width: 50px;
+  height: 4px;
+  margin-bottom: 5px;
+  position: relative; 
+  
+  background:var(--royalBlue);
+  border-radius: 3px;
+  z-index: 1;
+  bottom: 23px;;
+  
+}
+.span-3
+{
+  display: block;
+  width: 30px;
+  height: 4px;
+  margin-bottom: 5px;
+  position:relative;  
+  background:var(--royalBlue);
+  border-radius: 3px;
+  z-index: 1;
+  left:20px;
+  bottom: 16px;
+
+}
```

Looks like you're repeating the same styles in three places. We can change this to

```suggestion
.span-1, .span-2, .span-3
{
  display: block;
  width: 30px;
  height: 4px;
  margin-bottom: 5px;
  position: relative;
  
  background:var(--royalBlue);
  border-radius: 3px;
  z-index: 1;
  bottom: 30px;
  
  
}
```

---

----
---------
      ----- they are not exactly the same. different width and bottom position